parse/StringValueRefParser.cpp

```diff
@@ -43,6 +43,7 @@ namespace parse {
             |    tok.BuildingType_
             |    tok.Focus_
             |    tok.PreferredFocus_
+            |    tok.Hull_
```

Comment:
Is this doing something for this pull request...?

Reply:
it seems the merge went wrong in a big way

Reply:
Actually this one is ok, i need to access the fuel value of the hull for the effect which replaces the default hull fuel effect:
 
effects = SetMaxFuel value = ( Value + HullFuel name = Source.Hull )

Reply:
Could also have used `ThisHull`

Reply:
Ah, true

Reply:
but exposing Hull is also OK